packages/mwp-language-plugin/src/util/getLanguage.js

```diff
@@ -27,8 +27,17 @@ export const getCookieLang: ParseRequestLang = (request: HapiRequest) => {
  * get the URL-specified language - check for existing language path prefixes
  */
 export const getUrlLang: ParseRequestLang = (request: HapiRequest) => {
+	const { url, info: { referrer }, route } = request;
 	const { supportedLangs } = getServerSettings(request);
-	const urlLang = request.url.path.split('/')[1];
+	const { plugins = {} } = route.settings || {};
```

Comment:
I'm pretty sure `route.settings` will always be an object, and `route.settings.plugins` gets initialized to `{}` - no need for fallbacks here

I think `route.settings` is now just an alias to the (preferred) `route.options` object https://hapijs.com/api#-routeoptionsplugins